src/tools/yarprobotstatepublisher/src/main.cpp

```diff
@@ -35,9 +35,11 @@ int main(int argc, char *argv[])
     if (rf.check("help"))
     {
         cout<<"Options"<<endl;
-        cout<<"\t--robot                <robot-name>: robot name"<<endl;
+        cout<<"\t--namePrefix             <name-prefix>: prefix of the yarprobotstatepublisher ports"<<endl;
```

Comment:
The style of the existing options seems to be dash, separated, so I guess for consistency it would make sense to have all these options to be `--name-prefix` (the docs indeed already refer to them as that).

Reply:
By the way, "name-prefix" is a bit ambiguous as an option. If we want to change from the even more ambigous `robot`, what about `robot-port-prefix` or something similar?

Reply:
> The style of the existing options seems to be dash, separated, so I guess for consistency it would make sense to have all these options to be `--name-prefix` (the docs indeed already refer to them as that).

Done.

> what about `robot-port-prefix` or something similar?

It is the prefix for all the ports opened by the `yarprobotstatepusblisher` (`/yarprobotstatepublisher`, `/yarprobotstatepublisher/transformClient/rpc:i`, `/yarprobotstatepublisher/transformClient/rpc:o`, `yarprobotstatepublisher/transformClient/transforms:i`), I don't see why to add the word `robot` to this option. However, if you think it can give more continuity to the previous name (`robot`) for me its fine.